pwa/src/components/logs/logTable.tsx

```diff
@@ -172,10 +197,11 @@ export default function LogTable({ id = null }) {
                     role="tabpanel"
                     aria-labelledby="logGeneral-tab"
                   >
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-3">Type: {log.type && log.type}</h5>
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced">Call id: {log.callId && log.callId}</h5>
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced">Reponse time: {log.responseTime && log.responseTime}</h5>
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced">Route: {log.routeName && log.routeName}</h5>
+                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-3">Type: {log.type === 'in' ? 'Incoming' : 'Outcoming'}</h5>
+                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-1">Call id: {log.callId && log.callId}</h5>
+                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-1">Reponse time: {log.responseTime && log.responseTime} seconds</h5>
+                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-1">Route: {log.routeName && log.routeName}</h5>
+                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-1">Status: {log.requestStatus && log.requestStatus} <b style={{ color: log.responseStatusCode > 199 && log.responseStatusCode < 300 ? 'limegreen' : 'red' }}>{log.responseStatusCode} <i style={{ color: 'limegreen' }} className="fas fa-circle"></i></b></h5>
```

Comment:
Ik denk dat dit een tabel moet worden i.p.v. dat het een aantal `<h5>` elementen zijn.

Ik zie een hoop `log.routeName && log.routeName`, wat is de reden voor deze syntax? Volgens mij zou enkel `log.routeName` hetzelfde doel bereiken.